src/LuaTranspiler.ts

```diff
@@ -119,11 +121,13 @@ export class LuaTranspiler {
                     // Graciously handle transpilation errors
                     console.error("Encountered error parsing file: " + exception.message);
                     console.error(`${sourceFile.fileName} (${1 + pos.line},${pos.character})\n${exception.stack}`);
+                    return 1;
```

Comment:
Might be nice to make these constants or an enum.

Reply:
I agree, though we should probably discuss error codes and general error handling of tstl in a separate PR.